config/schedule.yml

```diff
@@ -1,4 +1,3 @@
 ingest_latest_blocked_lane_cases:
-  cron: '0 5 * * *' # Daily at 5 AM
+  cron: '0 12 * * *' # Daily at 12 UTC
```

Comment:
The worker instance operates on UTC time so I've updated the schedule to run at roughly 5 AM PST daily (depending on daylight savings).